app/src/main/java/com/eveningoutpost/dexdrip/Models/Bubble.java

```diff
@@ -82,6 +82,11 @@ public static BridgeResponse decodeBubblePacket(byte[] buffer, int len) {
             patchUid = Arrays.copyOfRange(buffer, 2, 10);
             String SensorSn = LibreUtils.decodeSerialNumberKey(patchUid);
             PersistentStore.setString("LibreSN", SensorSn);
+            
+            if (SensorSanity.checkLibreSensorChangeIfEnabled(SensorSn)) {
+                Log.e(TAG, "Problem with Libre Serial Number - not processing");
+            }
```

Comment:
This logs that it isn't processing but still continues. Should the message be changed?